src/Mailgun/Connection/RestClient.php

```diff
@@ -147,6 +147,9 @@ public function post($endpointUrl, array $postData = [], $files = [])
                     $postDataMultipart[] = [
                         'name' => sprintf('%s[%d]', $key, $index++),
                         'contents' => $subValue,
+                        'headers' => [
+                            'Content-Disposition' => sprintf('form-data; name="%s"', $key),
```

Comment:
Im afraid that the argument "others do not follow standards or best practices" is not a very good one.